atlp-rwanda/hackers-ec-Fe
# #187419022 Implement Cart Management Functionality

#### What does this PR do?
This PR introduces cart management functionality, allowing buyers to add items to their cart and view the cart.

#### Description of Task to be completed:
- Implement the UI components for adding items to the cart (quantity input and "Add to Cart" button).
- Create a confirmation message that displays the added item's name and quantity.
- Ensure that the form for selecting quantity allows only positive integers.
#### How should this be manually tested?
1. Clone the repo and cd into it
2. Checkout to ft-cart-management-187419022 and run npm install
3. Run npm run dev to start the application locally
4. Navigate to a product page.
5. Select a quantity and click the "Add to Cart" button.
6. Verify that the item is added to the cart and that the confirmation message appears.
7. Test the "View Cart" feature by clicking the corresponding button
#### Pivot tracker Id
#187419022
